packages/editor/src/client/panel/userEditPanel.tsx

```diff
@@ -207,6 +210,23 @@ export function UserEditPanel({id, onClose, onSave}: UserEditPanelProps) {
                 <Button appearance="primary" onClick={() => setIsResetUserPasswordOpen(true)}>
                   {t('userList.panels.resetPassword')}
                 </Button>
+                <Button
+                  appearance="primary"
+                  style={{marginLeft: '20px'}}
+                  disabled={!email || !active}
+                  onClick={async () => {
```

Comment:
Should it maybe be disabled here before result?

Reply:
@penk85 good idea but with such a small task maybe a bit over engineered.